### PR TITLE
chore:FPLA-2271 removal of final care order

### PR DESCRIPTION
**Pull request by nkoranne**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2271

### Change description ###

added the code for removal of final order 8 (index 7) as per customer request. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java [hunk 1]
```diff
@@ -38,7 +39,10 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             data.put("hearingDetails", removeHearingLinkedToCmo(caseData.getHearingDetails(),
                 caseData.getSealedCMOs().get(1).getId()));
 
+
             data.put("sealedCMOs", removeCaseManagementOrder(caseData.getSealedCMOs()));
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java:43]: Make sure to remove the changes to CMOs

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java:43]: and the linked hearing above

Comment (Jamesnelson117) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java:43]: This is fine, DM was never actually performed on SA20C50002 yet. The DM is remove a CMO and to remove a final order.

**Review by Jamesnelson117 — APPROVED** [review]
LGTM

**Pull request merged**